src/sentry/replays/usecases/query/conditions/activity.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import annotations
+
+from snuba_sdk import Column, Condition, Function, Op
+
+from sentry.replays.usecases.query.conditions.base import ComputedBase
+
+
+class AggregateActivityScalar(ComputedBase[int]):
+    """Activity scalar condition class."""
+
+    @staticmethod
+    def visit_eq(value: int) -> Condition:
+        return Condition(aggregate_activity(), Op.EQ, value)
+
+    @staticmethod
+    def visit_neq(value: int) -> Condition:
+        return Condition(aggregate_activity(), Op.NEQ, value)
+
+    @staticmethod
+    def visit_gt(value: int) -> Condition:
+        return Condition(aggregate_activity(), Op.GT, value)
+
+    @staticmethod
+    def visit_gte(value: int) -> Condition:
+        return Condition(aggregate_activity(), Op.GTE, value)
+
+    @staticmethod
+    def visit_lt(value: int) -> Condition:
+        return Condition(aggregate_activity(), Op.LT, value)
+
+    @staticmethod
+    def visit_lte(value: int) -> Condition:
+        return Condition(aggregate_activity(), Op.LTE, value)
+
+    @staticmethod
+    def visit_in(value: list[int]) -> Condition:
+        return Condition(aggregate_activity(), Op.IN, value)
+
+    @staticmethod
+    def visit_not_in(value: list[int]) -> Condition:
+        return Condition(aggregate_activity(), Op.NOT_IN, value)
+
+
+def aggregate_activity() -> Function:
```

Comment:
This should be abstracted to a more generic module so the select statement and the filtering conditions can share the same code.  I'm trying to avoid a large diff.  You can see how well that's going lol